src/Organic/Organic.php

```diff
@@ -1337,15 +1351,8 @@ public function syncPluginConfig() {
                 $this->updateOption( 'organic::sentry_dsn', $sentryDSN, false );
                 $this->configureSentryForSite();
             }
-            if ( $config['triggerContentResync'] && ! $this->contentResyncTriggeredRecently() ) {
-                global $wpdb;
-                $wpdb->get_results(
-                    $wpdb->prepare(
-                        "UPDATE $wpdb->postmeta SET meta_value = 'unsynced' WHERE meta_key = %s",
-                        SYNC_META_KEY
-                    )
-                );
-                $this->updateContentResyncStartedAt();
+            if ( true === $config['triggerContentResync'] ) {
+                $this->triggerContentResync();
             }
         }
```

Comment:
Can we move the `if (! empty( $config ) ) { ...` up through these lines into the above try block?

Reply:
Updated. Not sure if this is exactly what you had in mind.

Reply:
Updated!